components/training/presenterBlock.tsx

```diff
@@ -55,7 +55,8 @@ export const PresenterBlock = ({ data }) => {
 const PresenterCard = ({ presenter, schema, index }) => {
   return (
     <div
-      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl md:h-full"
+      // eslint-disable-next-line tailwindcss/no-arbitrary-value
+      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl md:h-[680px] md:w-96"
```

Comment:
Move the arbitraty height to tailwind config

Reply:
done